src/UI/SlotArea.cpp

```diff
@@ -692,6 +692,9 @@ void cSlotAreaCrafting::ShiftClickedResult(cPlayer & a_Player)
 		// Broadcast the window, we sometimes move items to different locations than Vanilla, causing needless desyncs:
 		m_ParentWindow.BroadcastWholeWindow();
 
+		// added this to make sure that achievements are awarded
```

Comment:
The comment could use some refactoring. "Added this" has no real meaning in a comment, we don't want to track some situational development, we want to know what the code is doing.
A simple `// Award achievements:` would suffice in this case.

Reply:
@madmaxoft done

Reply:
Sorry for being a nit-picker, the comment should start with a capital letter.